### PR TITLE
create v8 announcement blog

### PR DESCRIPTION
**Pull request by hipstersmoothie**
Published PR with canary version: `8.0.0-canary.782.10316.0`

### CODE REVIEW

File: docs/blog/v8.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+   ```json
+   {
+     "labels": [{ "changelogTitle": "New Docs Yo!", "name": "documentation" }]
+   }
+   ```
+
+## Upload Assets - Globs
+
+You might want to upload multiple files or you might not know the file-name (ex: a vscode extension w/ a version number in the name), but you know the pattern of the filenames that I want to upload.
```
Comment (adierkens) [inline docs/blog/v8.md]:
```suggestion
You might want to upload multiple files or you might not know the file-name (ex: a vscode extension w/ a version number in the name), but you know the pattern of the filenames that you want to upload.
```

File: docs/blog/v8.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+image: https://images.unsplash.com/photo-1485795046599-702122cd1267?crop=entropy&cs=tinysrgb&fit=crop&fm=jpg&h=500&ixid=eyJhcHBfaWQiOjF9&ixlib=rb-1.2.1&q=80&w=1950
+title: Announcing "auto" 8.0.0
+author:
+  name: Andrew Lisowski
+  url: https://twitter.com/HipsterSmoothie
+  email: lisowski54@gmail.com
+---
+
+We are excited to announce the release of `auto` 8.0.0.
+This release brings a new command `next`, a new `shipit` use, more powerful label configuration, and more,
```
Comment (tylerkrupicka) [inline docs/blog/v8.md]:
```suggestion
This release brings a new command `next`, a new `shipit` use, more powerful label configuration, and more.
```

File: docs/blog/v8.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+If you don't know what `auto` yet, it's a CLI tool helps you version your code and automate any part of your release.
```
Comment (tylerkrupicka) [inline docs/blog/v8.md]:
```suggestion
If you are unfamiliar, `auto` is a CLI tool helps you version your code and automate any part of your release pipeline.
```

File: docs/blog/v8.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+}
+```
+
+Example: Run `auto next` from `beta` branch => Publish prerelease to `beta` tag.
+
+## Prerelease Branches with `shipit`
+
+The `shipit` command will now also publish a prerelease when ran from a prerelease branch. You can use this in a few different ways:
+
+1. 2 release branches: `master` and `next`
```
Comment (tylerkrupicka) [inline docs/blog/v8.md]:
```suggestion
1. Two release branches: `master` and `next`
```

File: docs/blog/v8.md (hunk 2 above)
Comment (adierkens) [inline docs/blog/v8.md]:
```suggestion
This release brings a new command `next`, a new use for `shipit`, more powerful label configuration, and more,
```

File: docs/blog/v8.md (hunk 3 above)
Comment (rans0m) [reply docs/blog/v8.md]: "is a CLI tool **which**" or is it **that**?

File: docs/blog/v8.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+# Or other OSs
+# Download a platform specific version of auto
+curl -vkL -o - https://github.com/intuit/auto/releases/download/v8.0.0/auto-linux.gz | gunzip > ~/auto
+# Make auto executable
+chmod a+x ~/auto
+```
+
+## `next` Command
+
```
Comment (jdroigk) [inline docs/blog/v8.md]: you say prerelease 10 times here.  maybe you can cut that back a bit

File: docs/blog/v8.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+This new command will make a prerelease of your code. This entails:
+
+1. Creates a prerelease on package management platform (supported by `npm` and `git-tag` plugins)
+2. Creates a tag and "Pre Release" on GitHub releases page.
+
+The only prerelease branch `auto` comes configured with is `next`.
+You can configure as many prerelease branches as you want with the [prereleaseBranches](../pages/autorc.md#prerelease-branches) `.autorc` option.
+Calling the `next` command from a prerelease branch will publish a prerelease, otherwise it will publish to the default prerelease branch.
```
Comment (jdroigk) [inline docs/blog/v8.md]: this is a confusing statement

**Comment by adierkens** [comment]
:rocket: PR was released in v8.0.0-next.18 :rocket:

**Comment by adierkens** [comment]
:rocket: PR was released in v8.0.0 :rocket:

**Pull request merged**